tests/components/template/test_weather.py

```diff
@@ -43,14 +53,24 @@ async def test_template_state_text(hass):
     await hass.async_block_till_done()
     hass.states.async_set("sensor.windspeed", 20)
     await hass.async_block_till_done()
+    hass.states.async_set("sensor.windbearing", "SE")
```

Comment:
This is minor but let's use a numeric value here. After all, bearing is an angle and many weather integrations actually return a number there.